be/src/olap/single_replica_compaction.cpp

```diff
@@ -112,23 +110,18 @@ Status SingleReplicaCompaction::_do_single_replica_compaction_impl() {
     Version proper_version;
     // 3. find proper version to fetch
     if (!_find_rowset_to_fetch(peer_versions, &proper_version)) {
-        LOG(WARNING) << _tablet->tablet_id() << " tablet don't need to fetch, no matched version";
-        return Status::Aborted("no matched version to fetch");
+        return Status::Cancelled("no matched versions for single replica compaction");
```

Comment:
What's the difference between Aborted and Cancelled?

Reply:
’Aborted‘  is a termination due to some kind of failure, ’Cancelled‘ is due to some normal reason.